includes/diff/DifferenceEngine.php

```diff
@@ -905,7 +922,9 @@ private function getRevisionHeader( Revision $rev, $complete = '' ) {
 
 			$msg = $this->msg( $title->userCan( 'edit', $user ) ? 'editold' : 'viewsourceold' )->escaped();
 			/* Wikia Change begin */
-			$header .= ' <span class="mw-rev-head-action">(' . Linker::linkKnown( $title, $msg, array(), $editQuery ) . ')</span>';
+			$header .= ' <span class="mw-rev-head-action">(' .
+				Linker::linkKnown( $title, $msg, [ 'data-action' => 'edit-revision-' . $type ], $editQuery ) .
```

Comment:
can we make a const of edit-revision- and list possible combinations for easier debugging?